content/snapshots.md

```diff
@@ -79,11 +79,11 @@ To schedule snapshots for all VMs in all deployments:
         snapshot_schedule: 0 0 7 * * * UTC
     ```
 
-1. Run `bosh deploy` to update your deployment.
+1. Run `bosh deploy` to update your Director deployment.
```

Comment:
`bosh create-env` would be the typical way to update the director. `bosh deploy` may make sense for some, if they have a director deploying another director.

Reply:
I fixed the command and added a link to it's detailed description.